src/rich-log/MessageBuilder.cc

```diff
@@ -1,38 +1,43 @@
 #include "MessageBuilder.hh"
 
+#include <cstdarg>
 #include <cstdio>
 
+#include <clean-core/assert.hh>
+
 using namespace rlog;
 
+void MessageBuilder::append_formatted(cc::string_view fmt, cc::span<cc::string const> args)
```

Comment:
This function is missing a `++i` in the if block, fixed in https://github.com/project-arcana/rich-log/pull/2

Reply:
fixed